nix/pkgs/io-engine/cargo-package.nix

```diff
@@ -91,6 +92,7 @@ let
       systemdMinimal.dev
       utillinux.dev
       rdma-core
+      keyutils
```

Comment:
note related to this PR:
somehow I wonder if the spdk-rs refactoring isn't complete yet, as we could be pulling dependencies from there rather than duplicating